tests/cases/fourslash/signatureHelpFilteredTriggerCharacters01.ts

```diff
@@ -0,0 +1,15 @@
+/// <reference path="fourslash.ts" />
+
+////function foo<T>(x: T): T {
+////  throw null;
+////}
+////
+////foo("/**/")
+
+goTo.marker();
+for (const triggerCharacter of ["<", "(", ","]) {
+    edit.insert(triggerCharacter);
+    verify.noSignatureHelpForTriggerCharacter(triggerCharacter);
+    edit.backspace();
+}
+verify.signatureHelp({ triggerCharacter: undefined });
```

Comment:
What does this verify?

Reply:
Signature help will fail if nothing is returned, but I'll make a more explicit version.